ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -10,26 +10,72 @@ include 'pan/types';
 
 type ssh_preferred_authentication = string with match(SELF, '^(gssapi-with-mic|hostbased|publickey|keyboard-interactive|password)$');
 
+type ssh_ciphers = string with is_valid_ssh_cipher(SELF);
+type ssh_hostkeyalgorithms = string with match(SELF, "^(ssh-(rsa|dss|ed25519)|ecdsa-sha2-nistp(256|384|521)|(ssh-rsa-cert-v01|ssh-dss-cert-v01|ecdsa-sha2-nistp256-cert-v01|ecdsa-sha2-nistp384-cert-v01|ecdsa-sha2-nistp521-cert-v01|ssh-rsa-cert-v00|ssh-dss-cert-v00|ssh-ed25519-cert-v01)@openssh.com)$");
```

Comment:
you can also group this more by adding ``(-cert-v01)?`` to the ecdsa-sha2-nistp part
ssh-ed25519-cert-v01 does not match or is this by design?